CONTRIBUTING.md

```diff
@@ -519,22 +505,22 @@ https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/maintain
 
    ```sh-session
    mpdev publish \
-         --deployer_image=gcr.io/YOUR-PUBLIC-PROJECT/COMPANY-IDENTIFIER/YOUR-APP/deployer:VERSION \
-         --gcs_repo=gs://YOUR-BUCKET/COMPANY-IDENTIFIER/YOUR-APP:TRACK
+         --deployer_image=gcr.io/YOUR-PUBLIC-PROJECT/COMPANY-IDENTIFIER/deployer:VERSION \
+         --gcs_repo=gs://YOUR-BUCKET/COMPANY-IDENTIFIER:TRACK
    ```
 
    For example:
 
    ```sh-session
    mpdev publish \
-         --deployer_image=gcr.io/conjur-cloud-launcher-onboard/cyberark/conjur-open-source/deployer:1.6.1 \
-         --gcs_repo=gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark/conjur-open-source:1.6
+         --deployer_image=gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:1.6.1 \
+         --gcs_repo=gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark:1.6
    ```
 
    You should see a published URL in the output, for example:
 
    ```sh-session
-   Version is available at gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark/conjur-open-source:1.6/1.6.1.yaml
+   Version is available at gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark:1.6/1.6.1.yaml
```

Comment:
same as above

Reply:
This is an example output, so I think it's okay to leave this one.